dashboard/app/models/rubric.rb

```diff
@@ -18,11 +18,16 @@ class Rubric < ApplicationRecord
   belongs_to :lesson
 
   def summarize
+    level_position = lesson.script_levels.filter {|sl| sl.levels.include?(level)}.first&.position
```

Comment:
Open to feedback on how to make this cleaner

Reply:
I've been nerd sniped! how about:
```suggestion
    script_level = lesson.script_levels.find {|sl| sl.levels.include?(level)}
```
and then use `script_level&.position` below?

Reply:
Oh that's much better thank you!

Reply:
hmm now that I'm looking at this again, should we raise if there is no script level? I'm not sure what good could come from silently allowing a rubric to have an invalid level / lesson combo, but I suppose i could see the argument that it could make tests less brittle

Reply:
I see you merged already. never mind, not important enough to open a new PR just for this

Reply:
It's in the back of my mind to add some assurances that the level of a rubric exists within the lesson, but we don't have it yet. I'd rather enforce that elsewhere, though. Just created [a task](https://codedotorg.atlassian.net/browse/AITT-148) for that